lib/src/components/NxLoadWrapper/__tests__/NxLoadWrapper.test.tsx

```diff
@@ -4,48 +4,87 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxLoadWrapper, { Props } from '../NxLoadWrapper';
-import NxLoadError from '../../NxLoadError/NxLoadError';
-import NxLoadingSpinner from '../../NxLoadingSpinner/NxLoadingSpinner';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+import NxLoadWrapper from '../NxLoadWrapper';
 
 describe('NxLoadError', function() {
   const children = <div className="children"/>,
-      retryHandler = jest.fn(),
-      getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxLoadWrapper, { children, retryHandler });
+      retryHandler = jest.fn();
+  const renderEl = rtlRenderElement(NxLoadWrapper, {children, retryHandler});
+  const quickRender = rtlRender(NxLoadWrapper, {children, retryHandler});
 
   it('renders a NxLoadError if there is an error', function() {
```

Comment:
In our RTL tests, test name should not be expressed in terms of child React components because RTL operates on the DOM level and can't test what child React components are involved

Reply:
> In our RTL tests, test name should not be expressed in terms of child React components because RTL operates on the DOM level and can't test what child React components are involved

Done 378dd78381908e038eeb8e1f9ad1696316e1488e